kodilla-jdbc/src/test/java/com/kodilla/jdbc/DbManagerTestSuite.java

```diff
@@ -38,6 +38,46 @@ public void testSelectUsers() throws SQLException {
         }
         rs.close();
         statement.close();
-        Assert.assertEquals(5, counter);
+
+        assertEquals(5, counter);
+    }
+
+    // Essential data and some metadata obtained from query result
+    @Test
+    public void testSelectUsersAndPostsAsStatement() throws SQLException {
+        //Given
+        DbManager dbManager = DbManager.getInstance();
+
+        //When
+        String sqlQuery = "SELECT U.FIRSTNAME AS FIRST_NAME, U.LASTNAME AS LAST_NAME, COUNT(U.ID) AS POSTS " +
```

Comment:
you used the same name for column and table. I would recommend to choose different name here